docker_explorer/explorer.py

```diff
@@ -65,16 +65,14 @@ def DetectDockerStorageVersion(self):
     """
     if not os.path.isdir(self.containers_directory):
       raise errors.BadStorageException(
-          'Containers directory {0} does not exist.'.format(
-              self.containers_directory))
+          f'Containers directory {self.containers_directory} does not exist.')
     container_ids_list = os.listdir(self.containers_directory)
     if not container_ids_list:
       raise errors.DockerExplorerError(
-          'Could not find any container in {0:s}.\n'
-          'Make sure the docker directory ({1:s}) is correct.\n'
-          'If it is correct, you might want to run this script'
-          ' with higher privileges.'.format(
-              self.containers_directory, self.docker_directory))
+          f'Could not find any container in {self.containers_directory}.\n'
+          f'Make sure the docker directory ({self.docker_directory}) is '
+          'correct.\nIf it is correct, you might want to run this script'
+          ' with higher privileges.')
```

Comment:
Nit ...

Maybe move the space to the end of the previous line to keep things consistent?